notebooks/geo/geoseries.ipynb

```diff
@@ -0,0 +1,244 @@
+{
```

Comment:
<p>I'd like to see another cell where we demonstrate the <code>.geo</code> accessor, too</p><p> </p><blockquote>Alternatively,  use the <code>.geo</code> accessor to access GeoSeries methods from a <code>bigframes.pandas.Series</code>object. </blockquote><blockquote><br></blockquote><blockquote><code>point_geom_series.geo.x</code></blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/googleapis/python-bigquery-dataframes/pull/1327/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>